source/docs/articles/sites/using-tmp-files.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Using Temporary Files
+description: Instructions for using temporary files in distributed environments.
+category:
+- getting-started
+keywords: tmp, temp files, tmp files, temporary files, mulitiple application containers, distributed environments
+---
+
+##Temporary File Management Across Multiple Application Containers
+
+If you’re using multiple servers to process requests, you’ll need to set up a temporary directory that is shared between all environments.
+
+To share the temporary directory between all environments, use: `sites/default/files/tmp`
+
+##DROPs  
+To manage your temporary files securely and reliably across DROPs when multiple server calls are required to process the files, add this to your code: `private://tmp`
```

Comment:
Unclear. add "this" doesn't say what `private://tmp` is or where to add it to the codebase. @timani